packages/office-ui-fabric-react/src/components/Slider/Slider.test.tsx

```diff
@@ -3,13 +3,22 @@ import * as ReactDOM from 'react-dom';
 import * as renderer from 'react-test-renderer';
 import * as ReactTestUtils from 'react-dom/test-utils';
 
-import { mount } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
 import { Slider } from './Slider';
 import { ISlider } from './Slider.types';
 import { ONKEYDOWN_TIMEOUT_DURATION } from './Slider.base';
 import { KeyCodes } from '../../Utilities';
 
 describe('Slider', () => {
+  let wrapper: ReactWrapper | undefined;
```

Comment:
This seems great because we're not creating so many new consts down below & ReactWrapper lets us do a full DOM render in these tests - any other benefits/reasons you chose this (just out of curiosity)?

Reply:
reason to have this declared here is so that we can unmount/clean it up after each test. see `afterEach` below. you can see a lot of similar pattern in our code base for other tests